### DISCUSSION TITLE
v2.0.0 stable release ETA?

### DISCUSSION THREAD

**Discussion by chahat-arora** [faisalman/ua-parser-js#654] [General]
When can we expect the stable version of the library with client hint APIs to be released in version v2.0.0?

**Comment by chahat-arora** [comment]
@faisalman any updates?

**Reply by faisalman** [reply]
As of today, no ETA yet, but hopefully soon :pray:

**Comment by faisalman** [comment]
I'll be releasing the stable version of v2.0 soon, hopefully by next week.
